tasks/databases.yml

```diff
@@ -5,4 +5,5 @@
     collation: "{{ item.collation | default('utf8_general_ci') }}"
     encoding: "{{ item.encoding | default('utf8') }}"
     state: "{{ item.state | default('present') }}"
+    target: "{{ item.state | default('') }}"
```

Comment:
If this just used `item.state`, then it wouldn't get a target dump file. This should be a different setting, like `item.target`, and it should `default(omit)`.